docs/examples/201/key-vault-secret-create/main.bicep

```diff
@@ -129,7 +129,7 @@ resource secret 'Microsoft.KeyVault/vaults/secrets@2019-09-01' = {
 /*
 TODO: Replace the first secret workaround above with this once we have loops
```

Comment:
Can you uncomment this code now then?

Reply:
Good pt (minus the extra newline).

---
In reply to: [582064628](https://github.com/Azure/bicep/pull/1521#discussion_r582064628) [](ancestors = 582064628)

Reply:
Coming in next commit.

---
In reply to: [582208509](https://github.com/Azure/bicep/pull/1521#discussion_r582208509) [](ancestors = 582208509,582064628)

Reply:
Done.